.changeset/happy-parents-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clack/prompts": minor
+---
+
+Added `groupSpacing` to control spacing between multi select groups.
```

Comment:
```suggestion
Adds a new `groupSpacing` option to grouped multi-select prompts. If set to an integer greater than 0, it will add that number of new lines between each group.
```